[0m[[0minfo[0m] [0mRunning main [0m
[0m[[0mdebug[0m] [0mWaiting for threads to exit or System.exit to be called.[0m
[0m[[0mdebug[0m] [0m  Classpath:[0m
[0m[[0mdebug[0m] [0m	/home/syth/scalaSkils/fourthIteration/99Problems/target/scala-2.10/classes[0m
[0m[[0mdebug[0m] [0m	/home/syth/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.10.1.jar[0m
[0m[[0mdebug[0m] [0mWaiting for thread run-main-1 to terminate.[0m
[0m[[31merror[0m] [0m(run-main-1) scala.MatchError: List('b, 'c, 'c, 'a, 'a, 'd, 'e, 'e, 'e, 'e) (of class scala.collection.immutable.$colon$colon)[0m
scala.MatchError: List('b, 'c, 'c, 'a, 'a, 'd, 'e, 'e, 'e, 'e) (of class scala.collection.immutable.$colon$colon)
	at main$.compress(main.scala:34)
	at main$.compress(main.scala:35)
	at main$.compress(main.scala:35)
	at main$.main(main.scala:55)
	at main.main(main.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at sbt.Run.invokeMain(Run.scala:72)
	at sbt.Run.run0(Run.scala:65)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:54)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Logger$$anon$4.apply(Logger.scala:90)
	at sbt.TrapExit$App.run(TrapExit.scala:244)
	at java.lang.Thread.run(Thread.java:744)
[0m[[0mdebug[0m] [0m	Thread run-main-1 exited.[0m
[0m[[0mdebug[0m] [0mInterrupting remaining threads (should be all daemons).[0m
[0m[[0mdebug[0m] [0mSandboxed run complete..[0m
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1653)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1653)
	at scala.Option.foreach(Option.scala:236)
	at sbt.BuildCommon$class.toError(Defaults.scala:1653)
	at sbt.Defaults$.toError(Defaults.scala:35)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$36$$anonfun$apply$37.apply(Defaults.scala:656)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$36$$anonfun$apply$37.apply(Defaults.scala:654)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42)
	at sbt.std.Transform$$anon$4.work(System.scala:64)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
	at sbt.Execute.work(Execute.scala:244)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[0m[[31merror[0m] [0m(compile:[31mrun[0m) Nonzero exit code: 1[0m
